ompi/mca/common/monitoring/Makefile.am

```diff
@@ -50,15 +54,7 @@ libmca_common_monitoring_la_LDFLAGS =  \
 libmca_common_monitoring_la_LIBADD = $(common_monitoring_LIBS)
 libmca_common_monitoring_noinst_la_SOURCES = $(headers) $(sources)
 
-# These two rules will sym link the "noinst" libtool library filename
-# to the installable libtool library filename in the case where we are
-# compiling this component statically (case 2), described above).
-V=0
-OMPI_V_LN_SCOMP = $(ompi__v_LN_SCOMP_$V)
-ompi__v_LN_SCOMP_ = $(ompi__v_LN_SCOMP_$AM_DEFAULT_VERBOSITY)
-ompi__v_LN_SCOMP_0 = @echo "  LN_S    " `basename $(component_install)`;
-
-all-local:
+all-local: $(component_noinst)
```

Comment:
Since we're now including `Makefile.ompi-rules`, I think you need to also change `OMPI_V_LN_SCOMP` to `OMPI_V_LN_S` everywhere.